pipeline_steps/common.groovy

```diff
@@ -30,13 +30,6 @@ def void create_workspace_venv(){
     # UG-613 change TMPDIR to directory with more space
     export TMPDIR="/var/lib/jenkins/tmp"
 
-    # UG-612 Install pip, setuptools, and wheel on the host
```

Comment:
As long as we avoid the centos slaves, the version of pip is known as it is installed in the long running slave deploy process: https://github.com/rcbops/rpc-gating/blob/master/playbooks/setup_jenkins_slave.yml#L101-L118